### PR TITLE
Switched to PSR-2

### PR DESCRIPTION
**Pull request by lolautruche**
Jenkins updated for ezrobot to test against PSR-2 as well.

Made with [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer):

`php-cs-fixer --level=psr2`

### CODE REVIEW

**Comment by lolautruche** [comment]
Review ping @dpobel @yannickroger @bdunogier @andrerom @pspanja @glye

**Comment by dpobel** [comment]
wow, I'll be lost next time I have to write some PHP ;-)

I think while at it you should update the README of ezcs or change the content of the ezrobot comments when we use PSR-2 (if that's possible, I can't remember how it's configured).

**Comment by lolautruche** [comment]
:smile: 

I'll update README on ezcs repo.

**Comment by lolautruche** [comment]
I don't understand this error... WTF

**Comment by andrerom** [comment]
+1 when all robots agree :p (any idea why it fails?)

**Comment by lolautruche** [comment]
@andrerom I have no idea

**Comment by lolautruche** [comment]
It's more than likely that phpcs on Jenkins is outdated

File: Tests/Twig/TwigYuiExtensionTest.php [hunk 1]
```diff
@@ -45,74 +43,75 @@ protected function getFixturesDir()
...
                 $this->callback(
-                    function( $params )
-                    {
-                        return is_array( $params ) && isset( $params['module'] );
+                    function ($params) {
+                        return is_array($params) && isset($params['module']);
                     }
                 )
-            )
+            );
```
Comment (andrerom) [inline Tests/Twig/TwigYuiExtensionTest.php]: could it be this line it is not liking? despite saying line 102?

**Comment by andrerom** [comment]
> It's more than likely that phpcs on Jenkins is outdated

it probably is, unsure how we installed it actually.

File: Tests/Twig/TwigYuiExtensionTest.php (hunk 1 above)
Comment (lolautruche) [inline Tests/Twig/TwigYuiExtensionTest.php]: No this was a try I did 10min ago

**Comment by lolautruche** [comment]
CS check with PHPCS is now OK on Jenkins. @ezrobot fails for another reason (tests not run on Jenkins, known error according to @dpobel).

So can I merge this?

**Comment by dpobel** [comment]
+1

after the merge, @StephaneDiot @mhyndle @yannickroger don't forget to rebase your branch on master otherwise ezrobot will get very angry ;)

**Comment by andrerom** [comment]
@dpobel could you maybe write a small story for next sprint to get to the bottom of missing deps on jenkins with IT? Not good that we have ci "down" for longer time.

**Pull request merged**